canonical-web-and-design/partners.ubuntu.com
# Remove public pages

## Done

Based on https://github.com/canonical-web-and-design/partners.ubuntu.com/issues/245
- Remove all public pages from the site to **only** keep the CMS part and the json endpoints

## QA

- `./run`
- http://0.0.0.0:8003/
- http://0.0.0.0:8003/admin
- Add a bunch of partners
- http://0.0.0.0:8003/partners.json
- They should appear here
- Navigate on the cms and play around with it it shoudl still work
- Try to access public pages. They should 404

## Issue / Card

Fixes #246
